packages/create-discord-bot/template/JavaScript/src/util/loaders.js

```diff
@@ -37,17 +37,17 @@ export async function loadStructures(dir, predicate, recursive = true) {
 
 	// Loop through all the files in the directory
 	for (const file of files) {
-		// If the file is index.js or the file does not end with .js, skip the file
-		if (file === 'index.js' || !file.endsWith('.js')) {
-			continue;
-		}
-
 		// Get the stats of the file
 		const statFile = await stat(new URL(`${dir}/${file}`));
 
 		// If the file is a directory and recursive is true, recursively load the structures in the directory
 		if (statFile.isDirectory() && recursive) {
-			structures.push(...(await loadStructures(`${dir}/${file}`, predicate, recursive)));
+			structures.push(...(await loadStructures(new URL(`${dir}/${file}`), predicate, recursive)));
```

Comment:
since we use that `URL` twice now (here and 4 lines earlier) shouldn't it be defined as a const and be used twice for performance reasons?